app/controllers/api/v1/providers_controller.rb

```diff
@@ -7,20 +7,24 @@ class ProvidersController < ApplicationController
       # - delay between c# calculating datetime.UtcNow and value being written to postgres
       # - clock drift between servers
       def index
-        page_size = params[:per_page] || 100
-        ActiveRecord::Base.transaction do
-          ActiveRecord::Base.connection.execute('LOCK provider, provider_enrichment, site IN SHARE UPDATE EXCLUSIVE MODE')
-          @providers = Provider.changed_since(params[:changed_since]).limit(page_size)
-        end
-        last_provider = @providers.last
+        begin
+          page_size = params[:per_page] || 100
+          ActiveRecord::Base.transaction do
+            ActiveRecord::Base.connection.execute('LOCK provider, provider_enrichment, site IN SHARE UPDATE EXCLUSIVE MODE')
+            @providers = Provider.changed_since(params[:changed_since]).limit(page_size)
+          end
+          last_provider = @providers.last
 
-        response.headers['Link'] = if last_provider
-                                     next_link((last_provider.updated_at + 1.second).utc.iso8601, last_provider.id, page_size)
-                                   else
-                                     next_link(params[:changed_since], "", page_size)
-                                   end
+          response.headers['Link'] = if last_provider
+                                       next_link((last_provider.updated_at + 1.second).utc.iso8601, last_provider.id, page_size)
+                                     else
+                                       next_link(params[:changed_since], "", page_size)
+                                     end
 
-        render json: @providers
+          render json: @providers
+        rescue ActiveRecord::StatementInvalid => e
+          render json: { code: 400, status: 'Bad Request', message: 'Invalid changed_since value, the format should be a iso8601 timestamp' }.to_json, status: 400
```

Comment:
` render json: { code: 400, status: 'Bad Request', message: 'Invalid changed_since value, the format should be a iso8601 timestamp' }.to_json, status: :bad_request`



--

` status: :bad_request` vs  `status: 400`